src/mudlet-lua/lua/GUIUtils.lua

```diff
@@ -662,7 +672,13 @@ end
 --- @see bg
 --- @see showColors
 function fg(console, colorName)
-  local colorName = colorName or console
+  assert(type(console) == 'string', 'fg: bad argument #1 type (expected string, got '..type(console)..'!)')
```

Comment:
As above - need to say whether it is the window name or a colour name.

Reply:
True, but the need was there before already, nothing I introduced anew